imagedephi/redact/build_redaction_plan.py

```diff
@@ -35,7 +35,11 @@ def build_redaction_plan(
     file_format = get_file_format_from_path(image_path)
     if file_format == FileFormat.TIFF:
         # Since SVS is a subset of tiff, fall back on file extension
-        file_extension = FILE_EXTENSION_MAP[image_path.suffix]
+        file_extension = (
```

Comment:
As a follow on issue, we should detect Aperio files based on the value of the ImageDescription tag and not the svs suffix.